dance/conditional_parameter.yml

```diff
@@ -0,0 +1,133 @@
+#Search space for all preprocessing condition parameters
```

Comment:
Meaning is the processing order of FilterGenesScanpyOrder.The orders pointed to by order_index in FilterGenesScanpyOrder